packages/components/src/InputFile/InputFile.tsx

```diff
@@ -180,7 +196,18 @@ export function InputFile({
     options.accept = allowedTypes.join(",");
   }
 
-  const { getRootProps, getInputProps, isDragActive } = useDropzone(options);
+  const { getRootProps, getInputProps, isDragActive, fileRejections } =
+    useDropzone(options);
+  const validationErrors = fileRejections?.map(({ file, errors }) => {
+    return errors.map(error => {
+      return (
+        <InputValidation
+          message={file.name + "\n" + error.message}
```

Comment:
not sure how I feel about adding a newline character as the message to achieve the layout we want

I'll circle back on this

Reply:
wait, it's not showing up as a newline. what is this doing? just adding a space?

![image](https://github.com/GetJobber/atlantis/assets/11843215/55a3b45f-5566-45a8-bfcd-2941e56bf564)

Reply:
I think so I didn't dig into why that was the case though

Reply:
gotcha, alright if it's not doing the thing we'd expect it to do (newlining) - and we're ok with this result, meaning we don't need to find some other way to actually make it break to two lines - then let's remove the `"\n"` altogether and simply use a backticked string with some js vars to achieve the string format we want